src/components/List/ApplicationsList.tsx

```diff
@@ -81,45 +77,26 @@ interface ApplicationRowProps {
   index: number
   columns: Array<ColumnDetails>
   application: ApplicationListRow
-  handleExpansion: (row: ApplicationListRow) => void
 }
 
-const ApplicationRow: React.FC<ApplicationRowProps> = ({
-  index,
-  columns,
-  application,
-  handleExpansion,
-}) => (
-  <Table.Row
-    key={`ApplicationList-application-${index}`}
-    onClick={() => handleExpansion(application)}
-  >
-    {columns.map(({ headerName, ColumnComponent }) => (
-      <Table.Cell key={`ApplicationList-row-${index}-${headerName}`}>
-        <ColumnComponent application={application} />
-      </Table.Cell>
-    ))}
-    <Table.Cell icon="angle down" collapsing />
-  </Table.Row>
-)
-
-interface SectionsExpandedRowProps {
-  application: ApplicationListRow
-  index: number
-  colSpan: number
-}
+const ApplicationRow: React.FC<ApplicationRowProps> = ({ columns, application }) => {
+  const { push, query } = useRouter()
 
-const SectionsExpandedRow: React.FC<SectionsExpandedRowProps> = ({
-  application,
-  index,
-  colSpan,
-}) => {
-  const { serial } = application
   return (
-    <Table.Row key={`ApplicationList-application-${index}-sections`} colSpan={colSpan}>
-      <Table.Cell colSpan={colSpan}>
-        <Segment color="grey">TODO: SECTIONS</Segment>
-      </Table.Cell>
+    <Table.Row
+      key={`ApplicationList-application-${application.id}`}
+      onClick={() => {
+        console.log(query)
+        if (query.userRole === 'applicant') push(`/application/${application.serial}`)
+        else push(`/application/${application.serial}/review`)
```

Comment:
I don't really like how the whole row is clickable, especially since there is no cursor change to indicate that it will be. It's too easy to click and then get whisked away without wanting to. 
Possible solutions:
- make sure the cursor turns into the "clicking" icon on hover (usually achieved with `as="link"`)
- Only make the Title clickable, and make it a proper link (this would be my preferred option)
- Only make the little arrow on the right clickable

Reply:
Cool. I'll go with solution 2 (and 3) so we have the title clickable but also the right arrow that kind of indicate that will take you somewhere.

Reply:
I'm not happy about the looks of the right arrow - but now it's showing as clickable. I wonder how this will lock with actions in place. I'm thinking we might not want this anymore...

Reply:
Yeah, it seems to be a bit redundant now.